core/trino-main/src/test/java/io/trino/server/TestDynamicFilterService.java

```diff
@@ -304,9 +305,9 @@ public void testDynamicFilter()
         assertEquals(stats.getLazyDynamicFilters(), 3);
         assertEquals(stats.getReplicatedDynamicFilters(), 0);
         assertEquals(ImmutableSet.copyOf(stats.getDynamicFilterDomainStats()), ImmutableSet.of(
-                new DynamicFilterDomainStats(filterId1, getExpectedDomainString(1L, 2L), 2, 0),
-                new DynamicFilterDomainStats(filterId2, getExpectedDomainString(2L, 3L), 2, 0),
-                new DynamicFilterDomainStats(filterId3, Domain.none(INTEGER).toString(session.toConnectorSession()), 0, 0)));
+                new DynamicFilterDomainStats(filterId1, getExpectedDomainStatsString(1L, 2L, 2, INTEGER)),
```

Comment:
I think we know simplified domain here. Could we generate it and compare `toStrings`?